### ISSUE TITLE
No mixin for mask

### ISSUE THREAD

**Issue by pier-oliviert** [Compass/compass#646]
Right now, Compass doesn't provide any mixin for the mask rules. Is it planned on the roadmap?

**Comment by nickcooley** [comment]
@pothibo -- wondering, since masks are only available in Webkit (and not part of the CSS spec), what you would be looking for in a "mask" mixin? https://developer.mozilla.org/en/CSS/-webkit-mask

**Comment by pier-oliviert** [comment]
Been too long ago, can't remember, but if it's not in a standard, than I guess it's a no go for compass?

**Comment by nickcooley** [comment]
Yeah, I'm not sure that a mixin would add any value since there's only one syntax for entering masks...

**Comment by designbyadrian** [comment]
Don't mean to bump this, but firefox supports the standard mask property, while some webkit browsers support only the -webkit-mask proprety.

**Comment by nessup** [comment]
+1

**Comment by mechanicalduck** [comment]
+1

**Comment by lorenzodianni** [comment]
+1

**Comment by hjujah** [comment]
+1

**Comment by ajsharp** [comment]
+1

**Comment by joshudev** [comment]
+1

**Comment by arbaouimehdi** [comment]
+1

**Comment by pakMann** [comment]
+1

**Comment by JamesVanWaza** [comment]
@blackbiron 
@Arbaoui-Mehdi 
@joshudev 
@lorenzodianni 
@mechanicalduck 
@nessup 

You can now add +1 using the mentions tab.
